docs/user-guide/release-notes/2.0.x.rst

```diff
@@ -1,6 +1,19 @@
 2.0 Release Notes
 =================
 
+2.0.1
```

Comment:
PR is now https://github.com/pulp/pulp_docker/pull/151, and I think Redmine should be cleaned up now.